js/company.js

```diff
@@ -0,0 +1,91 @@
+async function getCompanyHistory(symbol) {
+    try {
+        const url =
+            "https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/api/v3/historical-price-full/" +
+            symbol + "?serietype = line";
+
+        const response = await fetch(url);
+        const result = response.json();
+
+        return result;
+    } catch (error) {
+        return false;
+    }
+}
+
+export async function getCompanyData(symbol) {
+    try {
+        const url =
+            "https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/api/v3/company/profile/" +
+            symbol;
+
+        const response = await fetch(url);
+        const result = response.json();
+
+        return result;
+    } catch (error) {
+        return false;
+    }
+}
+
+window.onload = async () => {
+    const companyImg = document.getElementById("company-img");
+    const companyName = document.getElementById("company-name");
+    const companyPrice = document.getElementById("company-price");
+    const companyPercentages = document.getElementById("company-percentages");
+    const companyDetails = document.getElementById("company-details");
+    const loader = document.getElementById("loader");
+
+    const urlParams = new URLSearchParams(window.location.search);
+    const symbolName = urlParams.get("symbol");
+
+    loader.classList.add('spinner-border');
+
+    const details = await getCompanyData(symbolName);
+    const companyProfile = details.profile;
+
+    companyImg.setAttribute("src", companyProfile.image);
+
+    companyName.setAttribute("href", companyProfile.website);
+    companyName.setAttribute("target", "_blank");
+    companyName.innerHTML = companyProfile.companyName;
+
+    companyPrice.innerHTML = "Stock price: " + companyProfile.price + "$";
+    companyPercentages.innerHTML = "(" + parseFloat(companyProfile.changesPercentage).toFixed(2) + "%)";
+
+    if (companyProfile.changesPercentage > 0) {
```

Comment:
Will it ever be negative? if not, you can use companyProfile.changesPercentage without the > 0.

That will be truethy and the code in the if statement will be executed